applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -222,6 +222,10 @@ KRATOS_CREATE_VARIABLE(Vector, EFFECTIVE_TENSION_STRESS_VECTOR)
 KRATOS_CREATE_VARIABLE(Vector, EFFECTIVE_COMPRESSION_STRESS_VECTOR)
 KRATOS_CREATE_VARIABLE(double, EXPONENTIAL_SATURATION_YIELD_STRESS)
 KRATOS_CREATE_VARIABLE(double, ACCUMULATED_PLASTIC_STRAIN)
+KRATOS_CREATE_VARIABLE(Vector, HIGH_CYCLE_FATIGUE_PARAMETERS)
```

Comment:
what is this used for? I thought Parameters are used to set the parameters in a CLaw?

Reply:
This is a vector with the parameters needed to create the Wöhler function. Probably it would be nicer to use HIGH_CYCLE_FATIGUE_COEFFICIENTS?

Reply:
Changed!